src/hearings/components/hearing-actual-summary/hearing-actual-summary.component.ts

```diff
@@ -39,6 +42,13 @@ export class HearingActualSummaryComponent implements OnInit {
     this.isPaperHearing$ = this.hearingState$ && this.hearingState$.pipe(
       map((state) => state.hearingRequest.hearingRequestMainModel.hearingDetails.hearingChannels.includes(HearingChannelEnum.ONPPR))
     );
+    this.hearingTypeDescription = hearingOutcome?.hearingType && this.getHearingTypeDescription(hearingOutcome.hearingType);
+  }
+
+  public getHearingTypeDescription(hearingType: string): string {
+    const hearingTypeFromLookup = this.hearingStageOptions?.find((x) => x.key.toLowerCase() === hearingType.toLowerCase());
```

Comment:
@SomeshRA you can change the access modifier of above function to private if you are not accessing from outside.
Also can you please add unit test for new code you have added in this PR otherwise it will fail in sonar cloud if unit tests are below 80% threshold limit.

Reply:
@RiteshHMCTS I have added tests for the new code